frameworks/CSharp/FastHttpApi/Benchmarks/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿using BeetleX.FastHttpApi;
+using Microsoft.Extensions.Hosting;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Threading;
+
+
+namespace Benchmarks
+{
+
+    [BeetleX.FastHttpApi.Controller]
+    class Program
+    {
+        private static string mDate;
+        public static void Main(string[] args)
+        {
+            mDate = DateTime.Now.ToUniversalTime().ToString("r");
```

Comment:
You can’t cache the date header either. It needs to update (we update it every second)